### ISSUE TITLE
Reset watched show in watch list (PS3-4)

### ISSUE THREAD

**Issue by Goten87** [UniversalMediaServer/UniversalMediaServer#2881]
i need some help with this back in the day i think i was able to reset watched eps with a folder/option in the list of the show i was watching on my ps3-4 and it went away with i think a setting i changed

my question is was it a setting or did you remove this function from the program if its still here what settting is it and how do i get it back (i dont want to reset my compleet database/settings)

**Comment by SubJunk** [comment]
Yes I think you're right we have lost the ability to do that on the renderer.
You can mark a folder as "fully played" in UMS itself still, by right-clicking on the folder.

**Comment by Goten87** [comment]
I don’t want to set it as fully played i want to remove that so I can rewatch it again also I cannot right click on a PlayStation

**Comment by SubJunk** [comment]
You can also mark it as "unplayed" with that right-click menu. But the limitation is that the GUI doesn't let you go into subfolders. It would be great if it did, but no one has made that yet.

You can change the name of the folder and then UMS will see them as new unplayed files

**Comment by SubJunk** [comment]
@Goten87 this can be done now in the web player
